sorc/ncep_post.fd/MISCLN.f

```diff
@@ -3724,15 +3725,18 @@ SUBROUTINE MISCLN
                   HELI(ista_2l:iend_2u,jsta_2l:jend_2u,2))
          allocate(LLOW(ista_2l:iend_2u,jsta_2l:jend_2u),LUPP(ista_2l:iend_2u,jsta_2l:jend_2u),   &
                   CANGLE(ista_2l:iend_2u,jsta_2l:jend_2u))
-
+         allocate(LLOW_ZINT(ista_2l:iend_2u,jsta_2l:jend_2u), 					 &
+                  IEQL_ZINT(ista_2l:iend_2u,jsta_2l:jend_2u),Z_TEMP(ista_2l:iend_2u,jsta_2l:jend_2u))
+         allocate(MIDCAL(ista_2l:iend_2u,jsta_2l:jend_2u,1:LM))
+         allocate(Z_MIDCAL(ista_2l:iend_2u,jsta_2l:jend_2u))
        iget1 = IGET(953)
        iget2 = -1
        iget3 = -1
        if (iget1 > 0) then
          iget2 = LVLS(1,iget1)
          iget3 = LVLS(2,iget1)
        endif
-      if(me==0) write(*,*) '953 ',iget1,iget2,iget3
+      if(me==0) write(0,*) '953 ',iget1,iget2,iget3
```

Comment:
@EdwardColon-NOAA Please revert your change. We would like to write non-fatal messages in stdout instead of stderr.